include/mbedtls/ssl_internal.h

```diff
@@ -149,7 +149,8 @@
     ( defined(MBEDTLS_AES_C)      ||                                  \
       defined(MBEDTLS_CAMELLIA_C) ||                                  \
       defined(MBEDTLS_ARIA_C)     ||                                  \
-      defined(MBEDTLS_DES_C) )
+      defined(MBEDTLS_DES_C)      ||                                  \
+      defined(MBEDTLS_BLOWFISH_C) )
```

Comment:
We have tests for record encryption/decryption using Blowfish-CBC:
```Record crypt, little space, BLOWFISH-CBC``` in `test_suite_ssl.c`
which fail if `MBEDTLS_SSL_SOME_SUITES_USE_TLS_CBC` doesn't include a case where only `MBEDTLS_BLOWFISH_C` is enabled.
Am I missing something here?

Reply:
I guess someone just added similar test cases for all the ciphers supported by `cipher.h` (and so indirectly by internal SSL functions that are driven by a cipher ID rather than by something parsed from a protocol message). The blowfish test cases should be removed from `test_suite_ssl.data`.

Reply:
Oh, right indeed I remember disagreeing at the time when Hanno added unit tests in `test_suite_ssl.data` that don't correspond to any real ciphersuite, but at that time it looked like it wouldn't hurt so I did not insist on removing them. Now that they do hurt, let's remove them.